modules/product/product.service.js

```diff
@@ -63,34 +65,20 @@ class ProductsService {
         },
       ];
     }
-    return Products.find(filters)
+    const items = await Products.find(filters)
       .skip(skip)
       .limit(limit)
       .sort(sort);
-  }
-
-  getProductsCount({ filter, search }) {
-    const isNotBlank = str => !(!str || str.trim().length === 0);
-    const filters = this.filterItems(filter);
 
-    if (isNotBlank(search)) {
-      filters.$or = [
-        {
-          name: { $elemMatch: { value: { $regex: new RegExp(search, 'i') } } },
-        },
-        {
-          description: {
-            $elemMatch: { value: { $regex: new RegExp(search, 'i') } },
-          },
-        },
-      ];
+    if (!items) {
```

Comment:
Does this check make any sense because we will always get an array from the database and it is always going to be a valid value